src/Knp/Rad/ResourceResolver/EventListener/ResourcesListener.php

```diff
@@ -105,4 +112,12 @@ protected function castParameter($parameter)
             }
         }
     }
+
+    protected function isNullValue($resource)
+    {
+        return null === $resource ||
```

Comment:
true for the collection maybe, but is it a common scenario?
Anyway, the condition can be simplified:

    if (empty($value) || isCountableObjectEmpty($value))